src/_realizeCustomElements.ts

```diff
@@ -241,9 +242,14 @@ function resolveOptions(registry: CombinedRegistry, element: Element, idFromAttr
 	return options;
 }
 
+function resolveStateFromAttribute(registry: CombinedRegistry, element: Element): Promise<StoreLike> {
+	const stateFrom = element.getAttribute('data-state-from');
+	return stateFrom ? registry.getStore(stateFrom) : null;
+}
+
 const noop = () => {};
 
-export default function realizeCustomElements(registry: CombinedRegistry, root: Element): Promise<Handle> {
+export default function realizeCustomElements(registry: CombinedRegistry, defaultStore: StoreLike, root: Element): Promise<Handle> {
```

Comment:
At some point, we should have JSDoc for all of our exported functions... even internal modules.  Ideally JSDoc for all of our functions.